go/vt/vttablet/tabletmanager/rpc_vreplication.go

```diff
@@ -62,6 +62,26 @@ const (
 	sqlUpdateVReplicationWorkflows = "update /*vt+ ALLOW_UNSAFE_VREPLICATION_WRITE */ %s.vreplication set%s where db_name = '%s'%s"
 	// Check if workflow is still copying.
 	sqlGetVReplicationCopyStatus = "select distinct vrepl_id from %s.copy_state where vrepl_id = %d"
+	// Validate the minimum set of permissions needed to manage vreplication metadata.
+	// This is a simple check for a matching user rather than any specific user@host
+	// combination.
+	sqlValidateVReplicationPermissions = `
+select if(count(*)>0, 1, 0) as good from mysql.user as u
```

Comment:
minor nit: `count(*)>0` already returns `1` or `0`. You may just as well:
```suggestion
select count(*)>0 as good from mysql.user as u
```